opm/simulators/flow/Main.cpp

```diff
@@ -119,6 +120,28 @@ Main::~Main()
 #endif
 }
 
+void Main::handleReservoirCouplingSlaveStdoutStderr_()
+{
+    // If first command line argument is "--slave-log-file=<filename>",
+    // then redirect stdout and stderr to the specified file.
+    if (this->argc_ >= 2) {
+        std::string_view arg = this->argv_[1];
```

Comment:
The assumption that if the first argument is `--slave-log-file` we can assume that this is a reservoir coupling slave, seems to be quite safe to me since a reservoir coupling slave can only be started by a master process. If this is not a slave process the user may still put `--slave-log-file` as the first argument by accident but that is not something that should happen very often.